chrome/locale/en-US/zotero/zotero.properties

```diff
@@ -1148,6 +1148,7 @@ lookup.failure.title				= Lookup Failed
 lookup.failure.description			= Zotero could not find a record for the specified identifier. Please verify the identifier and try again.
 lookup.failureToID.description		= Zotero could not find any identifiers in your input. Please verify your input and try again.
 lookup.failureTooMany.description	= Too many identifiers. Please enter one identifier and try again.
+lookup.failureMultiple.description	= Zotero could not find a record for the following identifier(s):\n\n%s\n\nPlease verify the identifier and try again.
```

Comment:
It's best to avoid newlines in messages, which are more likely to mangled during localization. Better to split into separate strings and hard-code the newlines in the code.

Also, we can just do plural, no "(s)".

And the second sentence should be plural.